runtime-modules/content/src/lib.rs

```diff
@@ -2304,6 +2308,23 @@ decl_module! {
             Self::deposit_event(RawEvent::NftBought(video_id, participant_id));
         }
 
+        /// Only Lead can toggle nft issuance limits constraints
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn toggle_nft_limits(
```

Comment:
This could result in unexpected behavior if there are multiple proposals created at a similar time to toggle this value. I think it should be explicit whether the proposal wishes to set the value to `true` or `false`, so the function signature should be `set_nft_limits_enabled(origin, enabled: bool)`. You also don't need to read the old value in that case, so it's more optimal.